src/manifold/src/boolean3.cpp

```diff
@@ -93,9 +93,9 @@ SparseIndices Filter11(const Manifold::Impl &inP, const Manifold::Impl &inQ,
                        const SparseIndices &p1q2, const SparseIndices &p2q1) {
   ZoneScoped;
   SparseIndices p1q1(3 * p1q2.size() + 3 * p2q1.size());
-  for_each_n(autoPolicy(p1q2.size()), countAt(0_z), p1q2.size(),
+  for_each_n(autoPolicy(p1q2.size(), 100'000), countAt(0_z), p1q2.size(),
```

Comment:
Can we make the policy not require e.g. `p1p2.size()`? I feel like the algorithm already has enough info to determine that. This is such a common pattern, it would be great to reduce it as much as possible. Could we make it work with e.g. `for_each_n({1e5}, countAt(0), ...)`?

Reply:
yes, I already updated things in par.h, should be easy to modify these as well.